src/snowflake/snowpark/modin/plugin/_internal/join_utils.py

```diff
@@ -1330,12 +1330,13 @@ def align_on_index(
     Args:
         left: Left DataFrame.
         right: right DataFrame.
-        how: the align method {{'left', 'coalesce', 'outer'}}, by default is outer
+        how: the align method {{'left', 'coalesce', 'outer', 'inner'}}, by default is outer
             * left: use only index from left frame, preserve left order.
             * coalesce: if left frame has non-zero rows use only index from left
                 frame, preserve left order otherwise use only right index and preserver
                 right order.
             * outer: use union of index from both frames, sort index lexicographically.
+            * inner: use intersection of index from both frames, preserve left order.
```

Comment:
Shouldn't there be a corresponding update to the body of this function?

Reply:
This method calls `align` which contains the implementation update for `how=inner` method